net/host_test.go

```diff
@@ -27,11 +27,11 @@ func (h *mockHandler) GetOffers() []*types.Offer {
 	return []*types.Offer{}
 }
 
-func (h *mockHandler) HandleInitiateMessage(_ *message.SendKeysMessage) (s SwapState, resp Message, err error) {
+func (h *mockHandler) HandleInitiateMessage(msg *message.SendKeysMessage) (s SwapState, resp Message, err error) {
 	if (h.id != types.Hash{}) {
-		return &mockSwapState{h.id}, &SendKeysMessage{}, nil
+		return &mockSwapState{h.id}, msg, nil
```

Comment:
This is a little hacky to reuse the same `SenkKeysMessage` like this.  Or at least, it might be confusing for someone learning the code by looking at the tests. Suggestions on how to do it better welcome.

Reply:
why does it need to do this? I guess it just needs to return a valid message? if so maybe making a valid test message might be more clear

Reply:
eg. return `createSendKeysMessage(t)`?

Reply:
That was what I wanted to do initially, but I didn't have access to `t`.  I added `t` to `mockHandler` so I could call it here.